Sources/BraveWallet/Crypto/NFT/NFTView.swift

```diff
@@ -50,25 +49,6 @@ struct NFTView: View {
     .padding(.horizontal, 32)
   }
   
-  private var editUserAssetsButton: some View {
```

Comment:
This wasn't in use for NFTs section anymore after updating to open Add NFT directly.